test/jdk/java/lang/Thread/virtual/stress/Skynet.java

```diff
@@ -26,7 +26,7 @@
  * @summary Stress test virtual threads with a variation of the Skynet 1M benchmark
  * @requires vm.continuations
  * @requires !vm.debug | vm.gc != "Z"
- * @run main/othervm/timeout=300 -Xmx1g Skynet
+ * @run main/othervm/timeout=300 -Xmx1500m Skynet
```

Comment:
Are these heap sizing changes to reduce the resource usage of this test or is it to try and trigger any potential issue that this test verifies?

Reply:
> Are these heap sizing changes to reduce the resource usage of this test or is it to try and trigger any potential issue that this test verifies?

The heap usage for this one can vary quite a bit, some of the lower optimization modes lead to more contention so there are are more FJP submissions. See JDK-8303635 for one example. It's been bumped to 1.5g in the loom repo for some time.

Reply:
>  It's been bumped to 1.5g in the loom repo for some time.

Ah, I misread that change previously as reducing it from `1g` to `150m`, while infact this change is increasing it to `1500m`. Given your reference to JDK-8303635, the change looks OK to me.